feature/platform/transceiver/tests/zr_logical_channels_test/zr_logical_channels_test.go

```diff
@@ -209,31 +225,60 @@ func validateOTNChannelTelemetry(t *testing.T, otnChIdx uint32, ethChIdx uint32,
 			got:  cc.GetAssignment(0).GetAssignmentType().String(),
 			want: oc.Assignment_AssignmentType_OPTICAL_CHANNEL.String(),
 		},
-		{
-			desc: "Ethernet Assignment: Index",
-			got:  cc.GetAssignment(1).GetIndex(),
-			want: uint32(1),
-		},
-		{
-			desc: "Ethernet Assignment: Logical Channel",
-			got:  cc.GetAssignment(1).GetLogicalChannel(),
-			want: ethChIdx,
-		},
-		{
-			desc: "Ethernet Assignment: Description",
-			got:  cc.GetAssignment(1).GetDescription(),
-			want: "OTN to ETH",
-		},
-		{
-			desc: "Ethernet Assignment: Allocation",
-			got:  cc.GetAssignment(1).GetAllocation(),
-			want: float64(400),
-		},
-		{
-			desc: "Ethernet Assignment: Type",
-			got:  cc.GetAssignment(1).GetAssignmentType().String(),
-			want: oc.Assignment_AssignmentType_LOGICAL_CHANNEL.String(),
-		},
+	}
+	var assignmentIndexTestcase struct {
```

Comment:
please consider previous comment to avoid lots of redeclaring this struct definition.